project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
   val slf4jVersion = "1.7.5"
   val awsVersion = "1.11.7"
   val faciaVersion = "2.0.2"
-  val capiModelsVersion = "9.24"
+  val capiModelsVersion = "10.1"
```

Comment:
The fact that these version numbers is so similar is in fact coincidental I think, they aren't always necessarily released together. @mchv probably knows more